docs/layers/grid-layer.md

```diff
@@ -62,6 +62,23 @@ Color scale domain, default is set to the range of point counts in each cell.
 Color ranges as an array of colors formatted as `[255, 255, 255]`. Default is
 [colorbrewer](http://colorbrewer2.org/#type=sequential&scheme=YlOrRd&n=6) `6-class YlOrRd`.
 
+##### `getColorValue` (Function, optional)
```

Comment:
Sounds like we are establishing a new proposed pattern here. Maybe add a section with proposed rules to [API GUIDELINES](https://github.com/uber/deck.gl/blob/master/API-GUIDELINES.md)